.github/dependabot.yaml

```diff
@@ -1,23 +1,20 @@
+# THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT.
+# Edit https://github.com/cert-manager/makefile-modules/blob/main/modules/repository-base/base/.github/dependabot.yaml instead.
```

Comment:
Broken link, I've fixed this upstream: https://github.com/cert-manager/makefile-modules/pull/196